OpenBlocks/src/main/resources/assets/openblocks/lang/en_us.json

```diff
@@ -12,6 +12,7 @@
     "block.openblocks.elevator.description": "Place one elevator directly three or more blocks above another with no blocks inbetween and you'll be able to either 'jump' to move up, or 'crouch' to move down. Elevators can be dyed by clicking on them with dye, however, you can only teleport between elevators of the same color.",
     "block.openblocks.rotating_elevator.name": "Rotating Elevator",
     "block.openblocks.heal": "Healer",
+    "item.openblocks.slimalyzer": "Slimalyzer",
```

Comment:
Please also grab [description](https://github.com/OpenMods/OpenBlocks/blob/master/src/main/resources/assets/openblocks/lang/en_us.lang#L325), it's not used yet, but will be for guide